tests/X86/DECIMAL/DAA.S

```diff
@@ -0,0 +1,370 @@
+/*
+ * Copyright (c) 2017 Trail of Bits, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#if ADDRESS_SIZE_BITS == 32
+
+#define CLEAR_PFSFZF  lahf; and ax, 0x3bff; sahf;
```

Comment:
Maybe extract these into Tests.S, and use them in your tests of `AAS`?

Reply:
Do you mean the defines?